inlong-manager/manager-client/src/test/java/org/apache/inlong/manager/client/api/inner/ClientFactoryTest.java

```diff
@@ -907,4 +910,32 @@ void testGetNode() {
         ClusterNodeResponse clientNode = clusterClient.getNode(1);
         Assertions.assertEquals(1, clientNode.getId());
     }
+
+    @Test
+    void testGetStreamSourceById() {
+        StreamSource streamSource = MySQLBinlogSource.builder()
+                .id(1)
+                .inlongGroupId("test_group")
+                .inlongStreamId("test_stream")
+                .sourceType(SourceType.MYSQL_BINLOG)
+                .sourceName("test_source")
+                .agentIp("127.0.0.1")
+                .inlongClusterName("test_cluster")
+                .user("root")
```

Comment:
Please use `.user("root").password("pwd")` instead of `root`.

Reply:
Done.